common/lib/capa/capa/capa_problem.py

```diff
@@ -667,11 +668,13 @@ def find_answer_text(self, answer_id, current_answer):
                 answer_id=answer_id,
                 choice_number=current_answer
             ))
-            assert len(elems) == 1
-            choicegroup = elems[0].getparent()
-            input_cls = inputtypes.registry.get_class_for_tag(choicegroup.tag)
-            choices_map = dict(input_cls.extract_choices(choicegroup, self.capa_system.i18n, text_only=True))
-            answer_text = choices_map[current_answer]
+            if len(elems) == 1:
```

Comment:
we need to check all cases 
1. we have not choice found
2. we have 1 choice
3. we have more than 1 choices found
```
if len(elems) == 0:
     log.warning("Answer Text Missing for answer id: %s and choice number: %s", answer_id, current_answer)
    answer_text = 'Answer Text Missing'
elif  len(elems) == 0:
    {current implementation}
else
    log.warning("Multiple answers found for answer id: %s and choice number: %s", answer_id, current_answer)
    answer_text = 'Multiple answers found'
```

Reply:
Updated.